generate-protos.sh

```diff
@@ -72,4 +76,6 @@ $PROTOC \
   -I tmp/google-cloudevents/proto \
   $(find tmp/google-cloudevents/proto -name data.proto)
 
-echo "Done"
+echo "- Removing tmp/"
+rm -rf tmp
```

Comment:
I find that keeping the `tmp` directory around (but ignored by .gitignore) until the next run can be really useful. If something hasn't generated as you expected it to, it's useful to be able to see all the intermediate aspects.

Reply:
It's best practice to clean up unintended side-effects like the `tmp` directory we create.
We can always re-run the script or comment out the cleanup steps if debugging.